### ISSUE TITLE
Year in review

### ISSUE THREAD

**Issue by Robbe7730** [ZeusWPI/g2-archive#2]
If a project has been going for a set amount of time (e.g. one year), G2 can generate a review of the previous year.

**Comment by wschella** [comment]
Good idea. Do we only want this for root projects? Or when it has a certain 'size'?

**Comment by Robbe7730** [comment]
Maybe this could be something on request by the maintainter?

**Comment by wschella** [comment]
We could suggest it for root projects. BTW I think suggesting 'connections' between projects, issues, and branches is something we really want to do, as to not make much automatic mistake, but still be powerful.
